ibm/operator_collection_sdk/playbooks/molecule/create_redeploy_delete_local_operator/converge.yml

```diff
@@ -0,0 +1,123 @@
+            namespace: "{{ ocpnamespace }}"
+          spec:
+            targetNamespaces:
+              - "{{ ocpnamespace }}"
+            upgradeStrategy: Default
+
+    - name: Create Subscripton
+      kubernetes.core.k8s:
+        state: present
+        definition:
+          apiVersion: operators.coreos.com/v1alpha1
+          kind: Subscription
+          metadata:
+            name: "{{ subscription_name }}"
+            namespace: "{{ ocpnamespace }}"
+          spec:
+            channel: v2.2
+            installPlanApproval: Automatic
+            name: ibm-zoscb
+            source: ibm-zoscb-internal-catalog
+            sourceNamespace: openshift-marketplace
+            startingCSV: ibm-zoscb.v2.2.1
+
+    - name: Validate Subscription installed successfully
+      kubernetes.core.k8s_info:
+        api_version: operators.coreos.com/v1alpha1
+        kind: Subscription
+        name: "{{ subscription_name }}"
+        namespace: "{{ ocpnamespace }}"
+      register: subscription_results
+      until: "subscription_results.resources[0].status.installedCSV is defined and subscription_results.resources[0].status.currentCSV"
+      retries: 30
+      delay: 5
+
+    - name: Validate CSV installed successfully
+      kubernetes.core.k8s_info:
+        api_version: operators.coreos.com/v1alpha1
+        kind: ClusterServiceVersion
+        name: "{{ subscription_results.resources[0].status.installedCSV }}"
+        namespace: "{{ ocpnamespace }}"
+      register: csv_results
+      until: "csv_results.resources[0].status.phase is defined and csv_results.resources[0].status.phase == 'Succeeded'"
+      retries: 30
+      delay: 5
+
+    - name: Create ZosCloudBoker
+      kubernetes.core.k8s:
+        state: present
+        definition:
+          apiVersion: zoscb.ibm.com/v2beta1
+          kind: ZosCloudBroker
+          metadata:
+            name: "{{ zoscloudbroker_name }}"
+            namespace: "{{ ocpnamespace }}"
+          spec:
+            catalogResources: {}
+            license:
+              accept: true
+            logLevel: trace
+            managerResources: {}
+            storage:
+              configure: false
+              enabled: false
+              size: 5Gi
+              volumeMode: Filesystem
+            uiResources: {}
+
+    - name: Validate ZosCloudBroker installed successfully
+      kubernetes.core.k8s_info:
+        api_version: zoscb.ibm.com/v2beta1
+        kind: ZosCloudBroker
+        name: "{{ zoscloudbroker_name }}"
+        namespace: "{{ ocpnamespace }}"
+      register: zoscb_results
+      until: "zoscb_results.resources[0].status.phase == 'Successful'"
+      retries: 30
+      delay: 5
+
+
+- import_playbook: ../../create_operator.yml
+  vars:
+    filepath: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') }}/../../../examples/racf-operator"
+    ocpnamespace: "{{ ocpNamespace | lower | replace('_','-') | replace('/','-') }}"
+
+- import_playbook: ../../redeploy_operator.yml
```

Comment:
The redeploy playbooks can be removed here since these endpoints are only created when running the `create_operator.yml` playbook

Reply:
@zohiba Yes, since neither of those playbooks would affect the endpoints....and those playbooks are already executed in another scenario for validation